develop/smart-contracts/block-explorers.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Block Explorers
+description: TODO
+---
+
+# Block Explorers
+
+## Introduction
+
+Block explorers serve as comprehensive blockchain analytics platforms that provide access to on-chain data. These web applications function as search engines for blockchain networks, allowing users to query, visualize, and analyze blockchain data in real-time through intuitive interfaces.
+
+## Core Functionality
+
+These block explorers provide essential capabilities for interacting with smart contracts in Polkadot Hub:
+
+- **Transaction Tracking** - monitor transaction status, confirmations, fees, and metadata
+- **Address analysis** - view account balances, transaction history, and associated contracts
+- **Block information** - examine block contents
+- **Smart contract interaction** - review contract code, verification status, and interaction history
+- **Token tracking** - monitor ERC-20/ERC-721/other token standards with transfer history and holder analytics
+- **Network statistics** - access metrics on transaction volume, gas usage, and other network parameters
+
+## Available Block Explorers
+
+The following block explorers are available for PolkaVM smart contracts, providing specialized tools for monitoring and analyzing contract activity within the Polkadot ecosystem:
+
+### Subscan
+
+Subscan is the flagship Polkadot ecosystem block explorer with comprehensive support for PolkaVM smart contracts. It provides robust transaction indexing, account analysis, and monitoring tools specifically optimized for Polkadot SDK-based networks.
+
+URL: [https://assethub-westend.subscan.io/](https://assethub-westend.subscan.io/){target=\_blank}
```

Comment:
This should be a tabbed thing per network? or not?

Reply:
Yeah that would be the best approach